### PR TITLE
[HIPIFY][SWDEV-316762][fix] Soft link to /opt/rocm/bin is missed for hipify-perl

### PR DESCRIPTION
**Pull request by aakanksha555**
Added a soft link for hipify-perl from /opt/rocm/bin to /opt/rocm/hip/bin

### CODE REVIEW

File: packaging/hipify-clang.rpm_postun [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ROCMDIR="/opt/rocm"
+HIPDIR=$ROCMDIR/hip
+HIPBINDIR=$HIPDIR/bin
```
Comment (emankov) [inline packaging/hipify-clang.rpm_postun:3]: HIPBINDIR=/opt/rocm/bin

**Review by emankov — COMMENTED** [review]
@aakanksha555, have you tested it on those OSs?

**Comment by aakanksha555** [comment]
> @aakanksha555, have you tested it on those OSs?

Tested the .deb packages.

**Comment by aakanksha555** [comment]
Do we want to package all the other scripts in bin as well? 

These a part of rocm 4.5 -
lrwxrwxrwx 1 root root        36 Dec  4 14:14 hipconvertinplace-perl.sh -> ../hip/bin/hipconvertinplace-perl.sh
lrwxrwxrwx 1 root root        31 Dec  4 14:14 hipconvertinplace.sh -> ../hip/bin/hipconvertinplace.sh
]lrwxrwxrwx 1 root root        29 Dec  4 14:14 hipexamine-perl.sh -> ../hip/bin/hipexamine-perl.sh
lrwxrwxrwx 1 root root        24 Dec  4 14:14 hipexamine.sh -> ../hip/bin/hipexamine.sh
lrwxrwxrwx 1 root root        22 Dec  4 14:14 hipify-perl -> ../hip/bin/hipify-perl

**Comment by emankov** [comment]
> Do we want to package all the other scripts in bin as well?
> 
The reported issue was about the soft link to hipify-perl, so the `-perl.sh` scripts might also be needed. As for the scripts without `-perl`, they are hipify-clang based. If you are going to provide the soft links to them, you should provide the soft link to `hipify-clang` too.

**Comment by emankov** [comment]
> > @aakanksha555, have you tested it on those OSs?
> 
> Tested the .deb packages.

I believe the fix has to be tested against all the target OSs.

File: packaging/hipify-clang.rpm_postun (hunk 1 above)
Comment (aakanksha555) [reply packaging/hipify-clang.rpm_postun:3]: Sure, can do that. Was just keeping up with convention in https://github.com/ROCm-Developer-Tools/hipamd/blob/develop/packaging/hip-devel.postinst

**Comment by aakanksha555** [comment]
> > Do we want to package all the other scripts in bin as well?
> 
> The reported issue was about the soft link to hipify-perl, so the `-perl.sh` scripts might also be needed. As for the scripts without `-perl`, they are hipify-clang based. If you are going to provide the soft links to them, you should provide the soft link to `hipify-clang` too.

Sure, I think we should do that. I believe we need coverage for all the components of rocm4.5 package in 5.0 as well.

**Pull request closed without merging**